<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageJingleDialog</class>
 <widget class="QDialog" name="MessageJingleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Jingle</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="comboLayout">
       <item>
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>Avant les messages d'ouverture :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ouvertureComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Avant les messages de fermeture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="femetureComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Avant les messages de promotion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="promotionComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Avant les messages programm√©s</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="programmesComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Avant l'envoi d'un message</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="envoiComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Enregister</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeSaveButton">
         <property name="text">
          <string>Enregistrer et fermer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageJingleDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageJingleDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
